lib/pbench/test/functional/server/conftest.py

```diff
@@ -24,7 +24,7 @@ def server_client():
     return client
 
 
-@pytest.fixture
+@pytest.fixture(scope="module")
```

Comment:
`DEBUG urllib3.connectionpool:connectionpool.py:456 http://localhost:8080/ "POST /api/v1/register HTTP/1.1" 403 52`

We are getting 403 on register that's bizarre and why did not having the module scope before didn't failed the test?